cookbooks/app-logs/recipes/default.rb

```diff
@@ -15,6 +15,14 @@
     recursive true
   end
 
+  directory "create shared directory to symlink log" do
```

Comment:
This is not required. Dir skeleton is created here: https://github.com/engineyard/ey-cookbooks-stable-v6/blob/next-release/cookbooks/app/recipes/create.rb#L2

Reply:
I moved `app-logs` before `app::create` https://github.com/engineyard/ey-cookbooks-stable-v6/pull/79/files#diff-71ecc5ebfa76a5140214e9870f6139a2 so I had to create the directory.

app-logs needs to run first or the php7.3-fpm package would fail to install.